### PR TITLE
tickets/DM-45821: Implement MT Dome slewing in MTCS

### PR DESCRIPTION
**Pull request by cvillalon**

### CODE REVIEW

**Review by tribeiro — APPROVED** [review]
Looks good!

**Comment by cvillalon** [comment]
I've pushed some mock definitions that were missing and caused the hanging in `test_slew_dome_to`. I think it is solved now. You may want to take another look @tribeiro so I'll wait for your confirmation before merging.

**Pull request merged**
